### PR TITLE
Fix #425. Attach the resize handler to the workspace so it can be rem…

### LINKED ISSUE #425: Multiple resize event handlers.
Blockly commit: c2368107c54e
An `onresize` event handler remains after disposing workspace and another one is added on next `Blockly.inject()` (according to Chrome DevTools). As a result resizing does not work after second `Blockly.inject()`. Resizing works if I remove onresize handler manually before second injection.

Resize handlers still leak in latest commit 04d6d119c076, but this does not seem to affect resizing.

### PR DESCRIPTION
**Pull request by picklesrus**
…oved

when workspace.dispose() is called.

### CODE REVIEW

File: core/workspace_svg.js [hunk 1]
```diff
@@ -65,6 +65,12 @@ Blockly.WorkspaceSvg = function(options) {
 goog.inherits(Blockly.WorkspaceSvg, Blockly.Workspace);
 
 /**
+ * Wrapper function called when a resize event occurs.
+ * @type {!Array.<!Array>} Data that can be passed to unbindEvent_
```
Comment (NeilFraser) [inline core/workspace_svg.js]: Looks like it can be null.  Drop the first "!".

File: core/workspace_svg.js [hunk 2]
```diff
@@ -139,6 +145,14 @@ Blockly.WorkspaceSvg.prototype.scrollbar = null;
 Blockly.WorkspaceSvg.prototype.lastSound_ = null;
 
 /**
+ * Save resize handler data so we can delete it later in dispose.
+ * @param  {!Array.<!Array>} Data that can be passed to unbindEvent_
```
Comment (NeilFraser) [inline core/workspace_svg.js]: Remove one space after @param.

Comment (NeilFraser) [inline core/workspace_svg.js]: Add the name "handler" after type information.

**Comment by NeilFraser** [comment]
Three comments.  LGTM

**Comment by picklesrus** [comment]
All fixed. Will merge. Thanks!

**Pull request merged**